### PR TITLE
Update component prop names

### PR DESCRIPTION
**Pull request by JKarlavige**
## What are you changing in this pull request and why?
This updates our existing `Snippet` and `FAQ` component `src` prop name to `path`. The link checker in PR #3673 was flagging these components as broken links due to the `src` prop. After updating the prop name to `path`, this link check did not report those components as broken links.

[Workflow run from this update](https://github.com/dbt-labs/docs.getdbt.com/actions/runs/5467755332)

## Previews
The following pages use the `Snippet` or `FAQ` components:
- Uses Snippet on 1st paragraph: https://deploy-preview-3675--docs-getdbt-com.netlify.app/docs/introduction
- Has FAQ section: https://deploy-preview-3675--docs-getdbt-com.netlify.app/docs/build/jinja-macros#faqs
- Discourse header is Snippet: https://deploy-preview-3675--docs-getdbt-com.netlify.app/docs/build/incremental-models#questions-from-the-community
- FAQs: https://deploy-preview-3675--docs-getdbt-com.netlify.app/docs/build/seeds#faqs

### CODE REVIEW

**Review by john-rock — APPROVED** [review]
lgtm!

Curious do we know why it was flagging the `src` prop on those components and not the Lightbox component?

**Comment by JKarlavige** [comment]
> lgtm!
> 
> Curious do we know why it was flagging the `src` prop on those components and not the Lightbox component?

I'm not 100% sure, but i'm thinking it's due to the value for the Lightbox `src` prop being a valid path to an image. For example the Lightbox src below loads the image successfully when adding to the url: https://docs.getdbt.com/img/blog/2022-11-22-move-spreadsheets-to-your-dwh/fivetran-uploader-2.png
```
<Lightbox src="/img/blog/2022-11-22-move-spreadsheets-to-your-dwh/fivetran-uploader-2.png" title="Picking the matching date format from a list of options to convert them to a standardized format" />
```

For the FAQ and Snippet components, the src prop only holds the filename for where that content is stored, and adding that value to the url returns a 404.

**Comment by john-rock** [comment]
> > lgtm!
> > Curious do we know why it was flagging the `src` prop on those components and not the Lightbox component?
> 
> I'm not 100% sure, but i'm thinking it's due to the value for the Lightbox `src` prop being a valid path to an image. For example the Lightbox src below loads the image successfully when adding to the url: https://docs.getdbt.com/img/blog/2022-11-22-move-spreadsheets-to-your-dwh/fivetran-uploader-2.png
> 
> ```
> <Lightbox src="/img/blog/2022-11-22-move-spreadsheets-to-your-dwh/fivetran-uploader-2.png" title="Picking the matching date format from a list of options to convert them to a standardized format" />
> ```
> 
> For the FAQ and Snippet components, the src prop only holds the filename for where that content is stored, and adding that value to the url returns a 404.

Ah that would make sense!

**Comment by JKarlavige** [comment]
Hey @mirnawong1, wanted to loop you in on this. We are updating the `Snippet` and `FAQ` components to use a `path` prop rather than `src`, as the `src` keyword was causing issues with a link checker we're working to implement. 

I believe this PR updates the documentation noted within the repo, but if there's additional documentation you and your team have, feel free to update it there as well!

**Review by breezyfasano — APPROVED** [review]
lgtm! nice find!

**Comment by mirnawong1** [comment]
oh great to know, thanks so much @JKarlavige for the heads up! so that means anything we want to use an faq or snippet (hopefully now replaced by partials), we'll need to use `path`:

 so `<FAQ path="Project/exclude-table-from-freshness" />`

**Comment by JKarlavige** [comment]
> oh great to know, thanks so much @JKarlavige for the heads up! so that means anything we want to use an faq or snippet (hopefully now replaced by partials), we'll need to use `path`:
> 
> so `<FAQ path="Project/exclude-table-from-freshness" />`

Yes that is correct!

File: contributing/single-sourcing-content.md
Comment (dataders) [inline contributing/single-sourcing-content.md:1]: > you can also pass data into the partials using a prop component

perhaps now would be a good time to document this new feature?

Comment (JKarlavige) [reply contributing/single-sourcing-content.md:1]: I believe @mirnawong1 is already on top of this! Under the `How to use props to pass different content on multiple pages?` toggle in the [single-sourcing-content](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#partial-example) doc.

Comment (mirnawong1) [reply contributing/single-sourcing-content.md:1]: thanks ya'll! i'll make this a bit clearer on my end. apologies for the delay,  am heads down on semantic layer docs 🙏

**Pull request merged**